commands/team_users.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var TeamUsersCmd = &cobra.Command{
+	Use:   "users",
+	Short: "Management of team users",
+}
+
+var RemoveUsersCmd = &cobra.Command{
+	Use:        "remove [team] [users]",
+	Short:      "Remove users from team",
+	Long:       "Remove some users from team",
+	Example:    "  team remove myteam user@example.com username",
+	Deprecated: "please use \"archive\" instead",
```

Comment:
I am not sure that archiving users from a team is an obvious command.  We do not use this phrasing anywhere else in the product.  I would recommend not using the archive command/not deprecating the ``remove users from team``,  as remove is more obvious that you are removing one or more users from their access to the team.

Reply:
Good for me, changing